client/src/pages/signup.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Button, TextField, Paper, FormControl } from '@material-ui/core';
+import { Link } from "react-router-dom";
+import laptopPhoto from '../images/blue-shirt-at-laptop.png'
+import './signup.css'
+
+export default function Signup() {
+
+    const [firstName, setFirstName] = useState("");
+    const [lastName, setLastName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [confirmPassword, setConfirmPassword] = useState("")
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if (password === confirmPassword) {
+            axios.post('/signup', {
+                firstName: firstName,
+                lastName: lastName,
+                email: email,
+                password: password
+            })
+                .then(window.location = "/")
+        } else {
+            alert("Passwords do not match")
+        }
+    }
+
+    return (
+        <div className="main">
+            <Paper className="image-container" variant="outlined">
+                <img className="laptop-image" src={laptopPhoto} alt="" />
+            </Paper>
+            <div className="right-screen">
+                <div className="to-sign-in">
+                    <p>Already have an account?</p>
+                    <Link to="/signin"><Button variant="outlined">Sign In</Button></Link>
```

Comment:
something I think you can do here is
```javascript
<Button to="/signin" component={Link} variant="outlined">Sign In</Button>
```

Reply:
This made a part of my useStyles (& Button), targeting the button as a child of another class, not work. I tried making it both & Button and & Link, but the Button would not be targeted. I don't feel too strongly either way, but if it's all the same, this saves one extra className.